src/System Application/App/Agent/Interaction/Internal/AgentTaskBuilderImpl.Codeunit.al

```diff
@@ -40,7 +40,7 @@ codeunit 4310 "Agent Task Builder Impl."
         end;
 
         if SetTaskStatusToReady then
-            AgentTaskImpl.SetTaskStatusToReadyIfPossible(AgentTaskRecord);
```

Comment:
Can't we have a caller verification to see if it is Microsoft owned app and throw an error if no message builder is set on the lines above and throw an error if the message is not set? Shouldn't this be the simplest implementation?